src/chainstate/coordinator/mod.rs

```diff
@@ -0,0 +1,380 @@
+use std::collections::VecDeque;
+use std::thread;
+use std::sync::{Arc, Mutex};
+use std::sync::mpsc::{self, Sender};
+use std::process;
+
+use burnchains::BurnchainHeaderHash;
+use chainstate::burn::{BlockHeaderHash as HeaderHash, BlockSnapshot as BurnchainBlock};
+use chainstate::stacks::{StacksBlock as Block, TransactionPayload};
+use chainstate::stacks::db::{StacksHeaderInfo as HeaderInfo};
+use chainstate::stacks::events::{StacksTransactionReceipt as TransactionReceipt};
```

Comment:
Unless there's an unavoidable reason for doing so (like, e.g., a name collision), can you please use the original struct names?

Reply:
Ha, made a note about this when opening the PR, would love to get your opinion

Reply:
I'd very much prefer _not_ renaming well-established structs and methods.  It only creates extra mental overhead for us, without improving things for new readers.

Reply:
Issue created - https://github.com/blockstack/stacks-blockchain/issues/1732

Reply:
Just a reminder -- my approval on this PR is blocked on reverting these name changes.  This PR should not be addressing #1732 in any way.

Reply:
Addressed with https://github.com/blockstack/stacks-blockchain/pull/1704/commits/fbc867ab05a98fe304f4ecd895ba05e172368716